cms/templates/cms/toolbar/toolbar_javascript.html

```diff
@@ -34,7 +34,8 @@
             'publish': '{% filter escapejs %}{% trans "Are you sure you want to publish this page?" %}{% endfilter %}',
             'addPluginPlaceholder': '{% filter escapejs %}{% trans "Plugin will be added here" %}{% endfilter %}',
             'confirmDirty': '{% filter escapejs %}{% trans "You have unsaved changes." %}{% endfilter %}',
-            'confirmDirtyESC': '{% filter escapejs %}{% trans "Are you sure you want to abandon these changes?" %}{% endfilter %}'
+            'confirmDirtyESC': '{% filter escapejs %}{% trans "Are you sure you want to abandon these changes?" %}{% endfilter %}',
+            'errorLoadingEditForm': '{% filter escapejs %}{% trans "Could not load the form. Please check if the server is running." %}{% endfilter %}'
```

Comment:
"The form could not be loaded. Please check that the server is running correctly."